src/types/shapediver/appbuilder.ts

```diff
@@ -177,7 +177,7 @@ export interface IAppBuilderSettingsSession extends SessionCreateDto {
  * Settings for initializing an AppBuilder application. This defines the sessions to create.
  */
 export interface IAppBuilderSettings {
-    "version": "1.0",
+    version: "1.0",
```

Comment:
@mathieu-shapediver this `version` property is not output by the current version of the Interstice example, which causes the validation to fail and display an error. Anyway, you had planned to update the example to use the prototyped components, which will solve this problem.